### PR TITLE
notebooks: fix missing attributions in BM2

### PR DESCRIPTION
**Pull request by wd15**
Address #1495

Fix some mistaken attributions for the free energy terms in BM2. These are carry overs from the original benchmark paper.

TODO:

 - [x] k to kappa

### CODE REVIEW

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1500.surge.sh

**Comment by wd15** [comment]
@tkphd, @guyer, see [this to review](https://random-cat-1500.surge.sh/benchmarks/benchmark2.ipynb/#Free-energy-and-dynamics). Also reivewnb to see the diff.

File: benchmarks/benchmark2.ipynb [hunk 1]
```diff
@@ -173,13 +173,14 @@
...
     "In this benchmark problem, the free energy of the system is based on\n",
-    "the formulation presented in Ref. [ZHU][] and is expressed as\n",
+    "a more general formulation of that presented in Ref. [ZHU][]. Specifically, the free energy here includes the concentration gradient energy as outlined in [WBM1][] and is expressed as\n",
     "\n",
     "$$\n",
     "F=\\int_{V}\\left(f_{chem}\\left(c,\\eta_{1},...\\eta_{p}\\right)+\\frac{\\kappa_{c}}{2}|\\nabla\n",
     "c|^{2}+\\sum_{i=1}^{p}\\frac{\\kappa_{\\eta}}{2}|\\nabla\\eta_{i}|^{2}\\right)dV\n",
     "$$\n",
     "\n",
+    "where the concentration gradient energy is \n",
```
Comment (tkphd) [inline benchmarks/benchmark2.ipynb:183]: This clause is missing an expression and punctuation.

File: benchmarks/benchmark2.ipynb.raw.html [hunk 2]
```diff
@@ -209,12 +209,13 @@ <h2 id="Free-energy-and-dynamics">Free energy and dynamics<a class="anchor-link"
...
 stress the numerical solver while not making the problem intractable.</p>
 <p>In this benchmark problem, the free energy of the system is based on
-the formulation presented in Ref. <a href="http://dx.doi.org/10.1016/j.actamat.2004.02.032">ZHU</a> and is expressed as</p>
+a more general formulation of that presented in Ref. <a href="http://dx.doi.org/10.1016/j.actamat.2004.02.032">ZHU</a>. Specifically, the free energy here includes the concentration gradient energy as outlined in <a href="https://doi.org/10.1103/PhysRevE.47.1893">WBM1</a> and is expressed as</p>
 <p>$$
 F=\int_{V}\left(f_{chem}\left(c,\eta_{1},...\eta_{p}\right)+\frac{\kappa_{c}}{2}|\nabla
 c|^{2}+\sum_{i=1}^{p}\frac{\kappa_{\eta}}{2}|\nabla\eta_{i}|^{2}\right)dV
 $$</p>
-<p>where $\kappa_{c}$ and $\kappa_{\eta}$ are the gradient energy
+<p>where the concentration gradient energy is
```
Comment (tkphd) [inline benchmarks/benchmark2.ipynb.raw.html:217]: Missing expr & punc.

**Review by tkphd — CHANGES_REQUESTED** [review]

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1500.surge.sh

**Review by tkphd — APPROVED** [review]
LGTM!

**Review by guyer — APPROVED** [review]

**Pull request merged**